sirepo/package_data/static/js/silas.js

```diff
@@ -106,6 +106,10 @@ SIREPO.app.controller('SourceController', function (appState, frameCache, persis
              ? 'parameter' : '3d';
     };
 
+    self.simCompletionState = (completionState) => {
+        return completionState.replace("animation frames", "completed reports");
```

Comment:
It's been a while since I've used this code but I think you want to override the "completionState" field in the silas schema. Here is what you will be overriding from schema-common.json https://github.com/radiasoft/sirepo/blob/d15784a34527b04ae868836d42d8fee45ddfbfda/sirepo/package_data/static/json/schema-common.json#L497

Reply:
Here is an example from activait https://github.com/radiasoft/sirepo/blob/d15784a34527b04ae868836d42d8fee45ddfbfda/sirepo/package_data/static/json/activait-schema.json#L461

Reply:
nice, thanks. Ill change it